homeassistant/components/ifttt/__init__.py

```diff
@@ -29,14 +30,15 @@
 
 SERVICE_TRIGGER_SCHEMA = vol.Schema({
     vol.Required(ATTR_EVENT): cv.string,
+    vol.Optional(ATTR_TO): vol.Any([cv.string], cv.string),
```

Comment:
Use `vol.All(cv.ensure_list, [cv.string])`, this will do the list wrapping for you.